### ISSUE TITLE
A bunch of duplicate field errors ?

### ISSUE THREAD

**Issue by AltGr** [ocaml/opam-repository#7445]
```
[WARNING] duplicate fields in /tmp/ff/repo/default/packages/gedcom/gedcom.1.0.0/opam: { bug-reports, authors }
[WARNING] duplicate fields in /tmp/ff/repo/default/packages/javalib/javalib.2.3.2/opam: { bug-reports }
[WARNING] licence is an unknown field in /tmp/ff/repo/default/packages/javalib/javalib.2.3.2/opam.
[WARNING] duplicate fields in /tmp/ff/repo/default/packages/procord/procord.0.2.0/opam: { dev-repo }
[WARNING] duplicate fields in /tmp/ff/repo/default/packages/sawja/sawja.1.5.2/opam: { bug-reports }
[WARNING] licence is an unknown field in /tmp/ff/repo/default/packages/sawja/sawja.1.5.2/opam.
[WARNING] duplicate fields in /tmp/ff/repo/default/packages/sosa/sosa.0.1.0/opam: { dev-repo }
```

Seems camelus missed those ? From my tests it's caught by opam 1.2.1 but not by 1.2.0, and camelus is built on 1.2.2, so that's strange.

**Comment by samoht** [comment]
PR welcome :-)

**Comment by dbuenzli** [comment]
`gedcom` seem to be fine, `javalib` was corrected by  f8067aff47526eeb602a6f9, `procord` by 549e07974a2e9bec35aa59fb417fce4de8c532f3, `sawja` by 8bd9ec8ad0abb0f5cf894092f9fda932f4e7adbe and `seso` seems to be fine. 

This can be closed.
